paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RollbackToAction.java

```diff
@@ -51,7 +56,16 @@ public void run() throws Exception {
         }
 
         if (version.chars().allMatch(Character::isDigit)) {
-            table.rollbackTo(Long.parseLong(version));
+            if (isTimestamp != null && isTimestamp) {
```

Comment:
boolean

Reply:
Here may not be boolean, because this argument may be null.